plugins/modules/mso_tenant.py

```diff
@@ -182,6 +197,12 @@ def main():
                 mso.existing = mso.request(path, method="DELETE")
 
     elif state == "present":
+        # Convert sites and users
+        sites = mso.lookup_sites(module.params.get("sites"))
+        users = mso.lookup_users(module.params.get("users"))
```

Comment:
Can we have a scenario where there are no local users but only remote users? If so lookup_users() and lookup_remote_users() would both be executed for no reason performing the same operation?
(I might be missing something)

Reply:
added check if the values are not None.